lms/djangoapps/courseware/toggles.py

```diff
@@ -113,6 +113,30 @@
     WAFFLE_FLAG_NAMESPACE, 'optimized_render_xblock', __name__
 )
 
+# .. toggle_name: courseware.mfe_special_exams
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable special exams experience without
+#   redirecting students to LMS.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-4-29
+# .. toggle_target_removal_date: 2021-6-30
+# .. toggle_warnings: None
+COURSEWARE_MICROFRONTEND_SPECIAL_EXAMS = CourseWaffleFlag(
+    WAFFLE_FLAG_NAMESPACE, 'mfe_special_exams', __name__
+)
+
+
+def special_exams_is_active(course_key: CourseKey) -> bool:
```

Comment:
nit: can we name this so it's explicitly referring to if it's active in the MFE

Reply:
addressed, please check